tests/detection/test_map.py

```diff
@@ -24,6 +27,35 @@
 
 Input = namedtuple("Input", ["preds", "target"])
 
+with open("tests/detection/instance_segmentation_inputs.json") as fp:
```

Comment:
@Borda can you move this file to the bucket where we have the other data test files stored?

Reply:
yes :)